products/apache-kafka.md

```diff
@@ -35,31 +35,31 @@ releases:
     latestReleaseDate: 2023-05-26
 
 -   releaseCycle: "3.3"
-    releaseDate: 2017-07-28
+    releaseDate: 2022-09-28
```

Comment:
Looks like there is an issue on Kafka

Reply:
Relates to https://github.com/endoflife-date/release-data/commit/ff8c22f6a4d2cdc6f8f748ddae90f9b72b338d2e and https://github.com/endoflife-date/release-data/commit/4130f9f3f2119dcf51b41c883d4d4abb70a951a1